src/Google.Storage.V1/PredefinedAcl.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Storage.V1
+{
+    /// <summary>
+    /// Predefined access control lists (ACLs) that can be specified when creating or updating resources.
+    /// </summary>
+    public enum PredefinedAcl
```

Comment:
Well spotted - I thought I'd looked through them all, but I must have missed it.

In other cases where the enums are similar but not the same we're currently going by number rather than by name - that needs fixing.

I think it's probably okay to have a single PredefinedAcl, but specify that some values are only useful in some contexts.